.gitignore

```diff
@@ -10,9 +10,7 @@ artifacts/
 build/
 cache/
 dist/
-deployments/*/*/artifacts
-deployments/*/*/cache
-deployments/*/*/contracts
+deployments/.contracts
```

Comment:
do we need to update the files deleted in `yarn spider --clean`, too?

Reply:
Probably, didn't know about that command -- where is it?

Reply:
https://github.com/compound-finance/comet/blob/main/tasks/spider/task.ts#L5

Reply:
Will follow on with a remove proxies PR hat touches this file, this is getting to large